packages/components/addon/components/hds/app-footer/legal-links.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+export default class HdsAppFooterIndexComponent extends Component {
+  /**
+   * @param urlForSupport
+   * @type {string}
+   * @description The href value of the "Support" link
+   */
+  get urlForSupport() {
```

Comment:
in the "Status" this parameter is called `href`, here `url`.

Reply:
In the `StatusLink` I'm exposing an existing `@href` argument so didn't want to rename it. You told me to use "urlForBlah" naming format for these getters previously so I did.

Should I change them to "hrefForBlah" now?

Reply:
probably for consistency yes, even if they read less well